services/src/main/java/io/druid/cli/CliOverlord.java

```diff
@@ -200,7 +199,6 @@ private void configureRunners(Binder binder)
             biddy.addBinding("remote").to(RemoteTaskRunnerFactory.class).in(LazySingleton.class);
             binder.bind(RemoteTaskRunnerFactory.class).in(LazySingleton.class);
 
-            JacksonConfigProvider.bind(binder, WorkerSetupData.CONFIG_KEY, WorkerSetupData.class, null);
```

Comment:
er, yep, meant to keep this comment:

Can this be noted in the release notes as well?